pyquil/quil.py

```diff
@@ -771,7 +799,10 @@ def get_default_qubit_mapping(program):
     return {qp: Qubit(i) for i, qp in enumerate(qubits)}
 
 
-def address_qubits(program, qubit_mapping=None):
+@no_type_check
```

Comment:
Personally I think this is fine, but pining @karalekas to make sure he sees it.

I think it's possible to satisfy mypy here by splashing around a bunch of `cast`s, but it's ugly and probably not worth it imo, so I'm fine with just ignoring this function.